futurex_openedx_extensions/upgrade/models_switch.py

```diff
@@ -0,0 +1,26 @@
+from futurex_openedx_extensions.upgrade.utils import (
+    FX_CURRENT_EDX_PLATFORM_VERSION,
+    FX_EDX_PLATFORM_VERSION_PALM,
+    FX_EDX_PLATFORM_VERSION_REDWOOD,
+)
+
+
+if FX_CURRENT_EDX_PLATFORM_VERSION == FX_EDX_PLATFORM_VERSION_PALM:
```

Comment:
Why can't we have separate master branches for different releases ?

Reply:
we don't want to maintain two branches. We're need to make sure the extension runs smoothly on both version